pkg/setting/setting.go

```diff
@@ -1284,6 +1285,12 @@ func (cfg *Cfg) handleAWSConfig() {
 	if err != nil {
 		cfg.Logger.Error(fmt.Sprintf("could not set environment variable '%s'", awsds.AllowedAuthProvidersEnvVarKeyName), err)
 	}
+
+	cfg.AWSExternalId = awsPluginSec.Key("external_id").Value()
+	err = os.Setenv(awsds.GrafanaAssumeRoleExternalIdKeyName, awsPluginSec.Key("external_id").Value())
```

Comment:
awsds.GrafanaAssumeRoleExternalIdKeyName is undefined atm, I guess there is an update of the sdk missing here

Reply:
ah yeah, I've added it [here](https://github.com/grafana/grafana-aws-sdk/pull/89) but I suppose I have to release the sdk first! My bad thanks for the reminder!